HpToolsLauncher/Runners/AlmTestSetsRunner.cs

```diff
@@ -1146,14 +1149,15 @@ public override TestSuiteRunResults Run()
             }
             catch (Exception)
             {
-                Console.WriteLine("Empty target test set list");
+                ConsoleWriter.WriteErrLine("Empty target test set list");
+                Launcher.ExitCode = Launcher.ExitCodeEnum.Failed;
+                return null;
```

Comment:
This line might be not needed. Please check and remove it if it's redundant.

Reply:
Removed.